app/Models/Audit.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Audit extends Model
+{
+    use HasFactory;
+    protected $fillable = ['action_type', 'user_id', 'affected_product_id'];
```

Comment:
Is product_id a better name than: affected_product_id? Also, do we need the audit to apply to non-products? E.g., Images? Should we consider a Polymorphic relationship here as it would allow us to have the same audit table used across all models (transactions / users / images / products etc).